test_runner/performance/test_lfc_prefetch.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+import random
+import threading
+import time
+
+import pytest
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnv
+from fixtures.utils import USE_LFC
+
+
+@pytest.mark.timeout(1000)
+@pytest.mark.parametrize("n_readers", [1, 2, 4, 8])
+@pytest.mark.parametrize("n_writers", [0, 1, 2, 4, 8])
+@pytest.mark.skipif(not USE_LFC, reason="LFC is disabled, skipping")
+def test_lfc_prefetch(neon_simple_env: NeonEnv, n_readers: int, n_writers: int):
+    """
+    Test resizing the Local File Cache
+    """
+    env = neon_simple_env
+    endpoint = env.endpoints.create_start(
+        "main",
+        config_lines=[
+            "neon.max_file_cache_size=10MB",
+            "neon.file_cache_size_limit=10MB",
+            "effective_io_concurrency=100",
+            "shared_buffers=1MB",
+            "enable_bitmapscan=off",
+            "enable_seqscan=off",
+            "autovacuum=off",
```

Comment:
Could you please explain, what effect of autovacuum is undesired for this test, why it's disabled?

Reply:
Well, autovacuum is not actually undesired in this test.
My original intention was to use this test also to measure speed. 
And in this case autovacuum may add unintended fluktuations.
Except it there is no more sense in disabling autovacuum.

Reply:
If the primary purpose of the test is measuring performance, not detecting defects, I wonder how to track performance with it. The test duration is constant, so shouldn't it export some metrics?

Reply:
Done